lib/aws/active_job/sqs/configuration.rb

```diff
@@ -86,13 +114,22 @@ class Configuration
         #   Using this option, job_id is implicitly added to the keys.
 
         def initialize(options = {})
-          options[:config_file] ||= config_file if File.exist?(config_file)
-          options = DEFAULTS
-                    .merge(file_options(options))
-                    .merge(options)
-          set_attributes(options)
+          opts = env_options.deep_merge(options)
+          opts = file_options(opts).deep_merge(opts)
+          opts = DEFAULTS.merge(opts)
+
+          set_attributes(opts)
         end
 
+        # @api private
+        attr_accessor :queues, :threads, :backpressure,
```

Comment:
Why are these private? Alternatively, can we remove them?

Reply:
I've updated this - generically we need attr accessors for setting and reading the config values, but I don't want these to appear in documentation and don't want users to rely on them.